src/Text/Markdown/SlamDown/Parser/Inline.purs

```diff
@@ -34,6 +46,103 @@ consolidate (Cons (Str s1) (Cons (Str s2) is)) =
   consolidate (Cons (Str (s1 <> s2)) is)
 consolidate (Cons i is) = Cons i $ consolidate is
 
+someOf :: (Char -> Boolean) -> Parser String String
+someOf p = (S.fromCharArray <<< fromList) <$> some (satisfy p)
+
+manyOf :: (Char -> Boolean) -> Parser String String
+manyOf p = (S.fromCharArray <<< fromList) <$> many (satisfy p)
```

Comment:
We should probably move these to `purescript-parsing` really, I didn't notice them here previously.